src/components/ec-main-container/ec-main-container.vue

```diff
@@ -55,6 +61,8 @@ defineProps<MainContainerProps>();
 
 const slots = useSlots();
 
+const isBelowBreakPointSize = useMediaQuery('(max-width: 344px)');
```

Comment:
as far as I understand the mobile threshold is 320, why 344 is used here?

Reply:
This is when the button should be wrapped based on new guidelines: https://www.figma.com/file/R8pt5Ic85URRNSyTagcKBd/%F0%9F%91%A5-Beneficiaries-redesign?node-id=3097%3A201712&mode=dev

Reply:
it's better to title the breakpoint with more meaningfull name, e.g. `isMobileView`

Reply:
That is not a style guide I guess, for example for 460px we have different representation. As far as I remember we selected 768 and 320 as thresholds, the last one is for mobile view. Let's at least clarify it?